tools/retip/macros.xml

```diff
@@ -0,0 +1,37 @@
+<macros>
+    <token name="@VERSION@">1.0.0</token>
```

Comment:
Please check [IUC best practices](https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#tool-versions) for a more robust versioning approach.